practical/ProcessMining/group2/inductive_miner/src/dfg_visualisation.py

```diff
@@ -0,0 +1,33 @@
+
+from pm4py.visualization.dfg import visualizer as dfg_visualizer
+from practical.ProcessMining.group2.inductive_miner.src.algo import DirectlyFollowsGraph, EventLog
+
+
+
+#TODO add additional event logs and tests
+
+
+
+l1 = EventLog({'abcd': 3,   'acbd' : 4, 'abcefbcd': 2,  'abcefcbd' : 1, 'acbefbcd' : 2, 'acbefbcefcbd' : 1})
+l2 = EventLog({'bc': 3,   'cb' : 4, 'bcefbc': 2,  'bcefcb' : 1, 'cbefbc' : 2, 'cbefbcefcb' : 1})
+
```

Comment:
Why do we need hard coded events?

Reply:
This was just for testing purposes, it will be removed in further refactoring